vm/systemSmartContracts/delegation.go

```diff
@@ -1827,6 +1898,11 @@ func (d *delegation) getContractConfig(args *vmcommon.ContractCallInput) vmcommo
 		changeableServiceFee = "true"
 	}
 
+	checkCapOnReDelegate := "false"
```

Comment:
we are not being consistent when using flags. On L496 we use yes/no, here true/false. Let's standardize and use constants.

Reply:
this is a view function only and it is treated in the delegation dashboard. I can change all to true / false

Reply:
having true / false everywhere will be great :+1: